### PR TITLE
Add docs for console logging changes.

### PR DESCRIPTION
**Pull request by markstory**
Refs cakephp/cakephp#7442

### CODE REVIEW

**Comment by antograssiot** [comment]
I push a comment as a commit, and I hope it was not just an english skills issue on my side :blush:

**Comment by ravage84** [comment]
:+1:

File: en/console-and-shells.rst [hunk 1]
```diff
@@ -468,9 +469,15 @@ You can mark output as follows::
...
 
-You can control the output level of shells, by using the ``--quiet`` and ``--verbose``
-options. These options are added by default, and allow you to consistently control
-output levels inside your CakePHP shells.
+You can control the output level of shells, by using the ``--quiet`` and
+``--verbose`` options. These options are added by default, and allow you to
+consistently control output levels inside your CakePHP shells.
+
+The ``--quiet`` and ``--verbose`` options also control how logging data is
+output to stdout/stderr. Normally info and higher log messages are output to
```
Comment (dereuromark) [inline en/console-and-shells.rst]: outputted

Comment (markstory) [inline en/console-and-shells.rst]: Both work I think.

Comment (ravage84) [inline en/console-and-shells.rst]: Yeah, it seems. It sounded right in my ears anyway :smirk_cat: 

> Preterite
I outputted
I output

http://conjugator.reverso.net/conjugation-english-verb-output.html

Comment (dereuromark) [inline en/console-and-shells.rst]: :+1:

**Pull request merged**